event-management-frontend/src/components/eventCreateOrEdit/locationSection/SearchBar.tsx

```diff
@@ -24,36 +24,75 @@ interface Props {
   searchMarker: LatLngExpression[];
   setsearchMarker: (searchMarker: LatLngExpression[]) => void;
 }
+
+function getUserDataWithPromise(props: string) {
+  const searchString =
+    'https://api.openrouteservice.org/geocode/autocomplete?api_key=' +
```

Comment:
You should extract this url in a separate constanst file

Reply:
Fixed